Add Joining Matrix page #1080 (#1081)
decred/dcrdocs: docs/getting-started/joining-matrix-channels.md

> # Joining the Decred community at Matrix

Reviewer: ```suggestion
# Joining the Decred community on Matrix
```

---

> ## What is Matrix

Reviewer: ```suggestion
## What is Matrix?
```

> Matrix is a secure, open messaging platform popular with Decred contractors. With the help of bridge software Matrix users can communicate with users on other Decred platforms such as Discord. Matrix offers first party registration; no identifying information or third party service is required to sign up. Users should be aware however that some features, such as password reset, do require an email address. For more on Matrix’s security features and available clients, see the Matrix project homepage.

Reviewer: ```suggestion
Matrix is a secure, open messaging platform popular with Decred contractors. With the help of bridge software Matrix users can communicate with users on other Decred platforms such as Discord. Matrix offers first party registration; no identifying information or third party service is required to sign up. Users should be aware however that some features, such as password reset, do require an email address. For more on Matrix’s security features and available clients, see the [Matrix project homepage](https://matrix.org/).
```

To join Matrix, you’ll need to choose a client and a homeserver. Matrix supports numerous clients and homeservers, but for first-time users, we recommend the most well-supported Matrix client, Riot, and Decred’s homeserver. Using the Decred homeserver makes it easier to find Decred chat rooms, puts `decred.org` in your ID, and allows you to continue messaging should Decred’s homeserver become disconnected with other homeservers.

---

## Creating your account

> Download [Riot](https://about.riot.im/), an open-source Matrix client.

Reviewer: We should mention that downloading Riot is optional, it can also be used on the web either at https://riot.im/ or https://chat.decred.org/

Author: Changed it to "Access Decred's server using Riot..." and added both links to the web browser option.

![Download Riot client](/img/matrix/matrix-01-download.png)

Riot supports multiple platforms:

- Desktop
- iOS
- Android

>   -Google Play

Reviewer: ```suggestion
     - Google Play
```
Need four spaces here, and then a space after the hyphen. Same for the next line

  -F-Droid
- Web Browser

When creating you account, choose to use another homeserver and enter `https://matrix.decred.org`.

![Select Home Server](/img/matrix/matrix-02-select-home-server.png)

If you filled the email field, check your in-box to verify your account.

![Create Account](/img/matrix/matrix-03-create-account.png)

Once in, you can begin exploring the different channels using the "Explore" feature.

![Explore Rooms](/img/matrix/matrix-04-explore-rooms.png)

Most Decred projects have their own channel, joining areas you would like to contribute to is encouraged.

![Room List](/img/matrix/matrix-05-room-list.png)

Anyone can create new channels. Feel free to make one for your Decred related project or friend group.

![Create Room](/img/matrix/matrix-06-create-room.png)

> Decred is a community of builders. Price and trading discussions have its own #trading channel.

Reviewer: ```suggestion
Decred is a community of builders. Price and trading discussions have their own `#trading:decred.org` channel.
```
